### DISCUSSION TITLE
Hello from the 🧨 diffusers team

### DISCUSSION THREAD

**Discussion by sayakpaul** [open-mmlab/mmediting#1903] [General]
Hi there, 

I hope you're doing well. I am Sayak, one of the maintainers of Diffusers. 

In the coming July, Diffusers will complete an anniversary. To celebrate this occasion, we are planning to do a series of blog posts as our way to thank all the awesome projects like yours that use Diffusers, help improve the library, and are an inspiration for us and the community.

We love how `mmagic` uses Diffusers features. Would you be interested in writing one/two sentences about your project that we could include in the blog posts?

Let us know 🤗

Best,
Sayak and the Diffusers team at 🤗

Cc: @patrickvonplaten @pcuenca

**Comment by Z-Fran** [comment]
Hi, @sayakpaul @patrickvonplaten @pcuenca Sorry for the late reply. Here is our simple introduction:
MMagic (Multimodal Advanced, Generative, and Intelligent Creation) is an advanced and comprehensive Generative AI toolbox that provides state-of-the-art AI models (e.g., diffusion models powered by diffusers and GAN) to synthesize, edit and enhance images and videos. In MMagic, users can use rich components to customize their own models like playing with Legos and manage the training loop easily.

**Reply by sayakpaul** [reply]
Thanks much!
